src/components/Waveform/Waveform.tsx

```diff
@@ -573,6 +573,8 @@ export const Waveform = forwardRef<IWaveformRef, IWaveform>((props, ref) => {
     pauseRecord: pauseRecordingAction,
     stopRecord: stopRecordingAction,
     resumeRecord: resumeRecordingAction,
+    currentState: mode === 'static' ? playerState : recorderState,
+    playerKey: () => path,
```

Comment:
```suggestion
    playerKey: path,
```

We can do this also. Is there any particular reason that you have made that function not just a string?

Reply:
@dprevost-LMI  Can you please take care of this as well.

Reply:
> Is there any particular reason that you have made that function not just a string?
At the time, I was thinking that using a function would ensure to have the proper path if it changes, but I'm not sure anymore if it is true 😄